common/lib/capa/capa/templates/mathstring.html

```diff
@@ -1,8 +1,9 @@
+<%page expression_filter="h"/>
```

Comment:
I suspect this is a dead file (only reference to it is in a test). That could be verified in course graph, then the file deleted.

In any case, I wouldn't recommend changing this file.